tokens/blocket.se/semantic.yml

```diff
@@ -7,155 +7,136 @@ s:
     background:
       _: white
       hover: bluegray-100
-      active:
+      active: bluegray-200
+      selected:
         _: blue-50
         hover: blue-100
       disabled: bluegray-300
+      inverted: gray-900
       subtle:
         _: bluegray-50
         hover: bluegray-100
-        active:
-          _: bluegray-100
-          hover: bluegray-200
-        disabled: bluegray-300
+        active: bluegray-200
       primary:
         _: blue-600
         hover: blue-700
-        active:
-          _: blue-800
-          hover: blue-700
+        active: blue-800
+        subtle:
+          _: blue-50
+          hover: blue-100
+          active: blue-200
       positive:
         _: green-600
         hover: green-700
-        active:
-          _: green-600
-          hover: green-700
+        active: green-800
+        selected:
+          _: green-50
+          hover: green-100
         subtle:
           _: green-50
           hover: green-100
-          active:
-            _: green-100
-            hover: green-200
+          active: green-200
       negative:
         _: red-600
         hover: red-700
-        active:
-          _: red-600
-          hover: red-700
+        active: red-800
+        selected:
+          _: red-50
+          hover: red-100
         subtle:
           _: red-50
           hover: red-100
-          active:
-            _: red-50
-            hover: red-100
+          active: red-200
       warning:
         _: yellow-600
         hover: yellow-700
-        active:
-          _: yellow-600
-          hover: yellow-700
+        active: yellow-800
+        selected:
+          _: yellow-50
+          hover: yellow-100
         subtle:
           _: yellow-50
           hover: yellow-100
-          active:
-            _: yellow-50
-            hover: yellow-100
+          active: yellow-200
       info:
         _: aqua-600
         hover: aqua-700
-        active:
-          _: aqua-600
-          hover: aqua-700
+        active: aqua-800
+        selected:
+          _: aqua-50
+          hover: aqua-100
         subtle:
           _: aqua-50
           hover: aqua-100
-          active:
-            _: aqua-50
-            hover: aqua-100
+          active: aqua-200
       notification: red-600
 
     border:
       _: bluegray-300
-      hover: gray-500
-      active:
-        _: gray-700
-        hover: gray-800
+      hover: bluegray-400
+      active: bluegray-500
       disabled: bluegray-300
+      inverted: white
```

Comment:
I couldn't find this one in the spreadsheet.

Reply:
It will be introduced there as well 👍 😜